crates/networking/p2p/rlpx/initiator.rs

```diff
@@ -26,40 +31,32 @@ pub struct RLPxInitiator {
 
     /// The target number of RLPx connections to reach.
     target_peers: u64,
-    /// The rate at which to try new connections.
-    new_connections_per_lookup: usize,
 }
 
 impl RLPxInitiator {
     pub fn new(context: P2PContext) -> Self {
         Self {
             context,
-            initial_lookup_interval: Duration::from_secs(3),
-            lookup_interval: Duration::from_secs(5 * 60),
+            // We use the same lookup intervals as Discovery to try to get both process to check at the same rate
+            initial_lookup_interval: INITIAL_LOOKUP_INTERVAL,
+            lookup_interval: LOOKUP_INTERVAL,
             target_peers: 50,
-            new_connections_per_lookup: 5000,
         }
     }
 
     pub async fn spawn(context: P2PContext) {
         info!("Starting RLPx Initiator");
         let state = RLPxInitiator::new(context);
-        let mut server = RLPxInitiator::start_on_thread(state.clone());
-        let _ = server.cast(InMessage::LookForPeers).await;
+        let mut server = RLPxInitiator::start(state.clone());
```

Comment:
Reverted this as it was shown it is not affecting performance and avoids creating a new Tokio runtime for the new thread